guides/common/modules/proc_upgrading-project-in-place-using-leapp.adoc

```diff
@@ -201,6 +201,24 @@ endif::[]
 # journalctl -u leapp_resume -f
 ----
 
+ifdef::satellite[]
+. Complete the post-upgrade steps described in https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/upgrading_from_rhel_7_to_rhel_8/verifying-the-post-upgrade-state-of-the-rhel-8-system_upgrading-from-rhel-7-to-rhel-8[Verifying the post-upgrade state of the RHEL 8 system] in the _Upgrading from RHEL 7 to RHEL 8_ guide.
+endif::[]
+. If you require SELinux to be in enforcing mode, run the following command before changing SELinux to enforcing mode:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+ifdef::foreman-el[]
+# dnf reinstall foreman-selinux --disableplugin=foreman-protector
+endif::[]
+ifndef::foreman-el[]
```

Comment:
```suggestion
ifdef::katello,satellite,orcharhino[]
```

Reply:
Out of curiosity, why? The result will be identical (this chapter is not rendered for foreman-deb, which is the only remaining target).

Reply:
I thought about if for serveral moments and came to the same conclusion. However, in terms of readability, this looks like an improvement to me. But also feel free to ignore and merge right away. rest LGTM!